lib/plugins/ramdisk.js

```diff
@@ -14,12 +14,30 @@ const { symlinkSync } = require('fs');
 const { basename, join, resolve } = require('path');
 
 const isCwd = require('is-path-cwd');
-const readPkgUp = require('read-pkg-up');
+const escalade = require('escalade/sync');
 const rm = require('rimraf');
 const { WebpackPluginRamdisk } = require('webpack-plugin-ramdisk');
 
 const { PluginExistsError, RamdiskPathError } = require('../errors');
 
+function readPkgName() {
```

Comment:
```suggestion
const readPkgName = () => {
```

We should export this so it can be tested